Resources/Prototypes/Entities/Objects/Tools/gas_tanks.yml

```diff
@@ -184,6 +202,32 @@
       types:
         Blunt: 7.5
 
+- type: entity
+  parent: ExtendedEmergencyOxygenTank
```

Comment:
set DoubleEmergencyOxygenTank as parent and you can left only sprite item and clothing components

Reply:
pluh

Reply:
Sorry! I was trying to figure out which thing you were on about from the conversation page, got it wrong lol!